### ISSUE TITLE
A later version of Vagrant is installed.  Please remove this version first.  Setup will now exit.

### ISSUE THREAD

**Issue by adamlucas78** [hashicorp/vagrant#9331]
I'm trying to downgrade my Vagrant install to 1.9.7 from 2.0.1 because when I upgraded to 2.0.1 I lost the bash prompt after ssh'ing into my Vagrant box (ubuntu/trusty64) using Cygwin.

### Vagrant version
2.0.1 (uninstalled via Add/Remove Programs and removed .vagrant.d directory)

### Host operating system
Windows 10

### Guest operating system
N/A

### Expected behavior
The uninstall should have removed all traces of Vagrant 2.0.1 and I should be able to install an older version of Vagrant.

### Actual behavior
When trying to install both 2.0.0 and 1.9.7 I get the error message: A later version of Vagrant is installed.  Please remove this version first.  Setup will now exit.

### Steps to reproduce
1.  Install Vagrant 2.0.1
2.  Uninstall Vagrant 2.0.1
3.  Install any prior version of Vagrant (at least 2.0.0 or 1.9.7 were the two I tested)

In all of my tests I'm using the x86_64.msi installer.

I also tried reinstalling 2.0.1 and then uninstalling it again thinking something didn't get completed the first time it uninstalled, but still wasn't able to install a prior version.

**Comment by briancain** [comment]
@adamlucas78 - I recommend using the Vagrant msi installer to uninstall Vagrant. It should have a "remove" option when you start it up. Try that, and then install an older version of Vagrant.

**Comment by adamlucas78** [comment]
@briancain No luck when uninstalling via the msi.  I'm still getting the same error message.

**Comment by briancain** [comment]
I'm not sure if this matters, but did you try uninstalling with the 2.0.1 msi?

**Comment by adamlucas78** [comment]
Yes, I definitely made sure to match the version of the msi.  I tried the 2.0.1 msi and verified the installed version before I started.  I wanted to roll back due to a different bug but ended up finding a workaround for that so this is a much lower priority for me.  I'd imagine I'm not the only one who has hit this so I wanted to point it out.
